src/sas/qtgui/Perspectives/SizeDistribution/media/sizedistribution_help.rst

```diff
@@ -5,15 +5,274 @@
+scattered intensity expected from a system of the chosen shape with the
+solution distribution of sizes, must be minimized.  In other words, of all the
+distributions that would satisfy the $\chi^2$ constraint, we find the one with
+the maximum entropy. The size distribution problem however is too complex to be
+conducive to an analytical solution and so an iterative process is employed.
+Here we employ an  algorithm due to Skillings and Bryan [#SkillingsAndBryan]_
+as implemented in GSAS II [#GSAS]_.
+
+Using the Size Distribution Analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Load some data with the *Data Explorer.*
+
+Select a dataset and use the *Send To* button on the *Data Explorer* to load
+the dataset into the *Size Distribution* panel.
+
+INSERT IMAGE HERE!!!
+
+This will open the panel on the *Parameters* tab and plot the data to fit.
+The most important parameters to adjust at this point are the minimum and
+maximum diameters of the distribution. The calculation will only explore
+diameters in this range. It is important that the range not excessivly exceed
+the limits imposed by the $Q$ range being fit.
+
+If the data is on absolute scale and a quantitave volume fraction is desired the
+contrast factor between the domain of interest and the matrix must be set
+correctly. Otherwise, the value is not important and can be left alone.
+
+The number of bins sets the number of sizes within the size range that will be
+calculated.
+
+The models section default is usually appropriate. Currently only the ellipsoid
+model is implemented. This may be expanded in the future to include cylinders
+for example. The aspect ratio for the ellipsoid however can be changed. The
+default of 1 is for a sphere. An aspect ration greater than 1 will yield a
+prolate ellipsoid while a value smaller than 1 is for an oblate ellipsoid.
+
+.. Warning::
+   The Size Distribution analyis assumes the data is properly background
+   subtracted. The smaller sizes in particular will be very sensitive to that.
+   If this is not the case proceed to the options tab as described below and
+   ensure that the background subtraction is set correctly.
+
+At this point, one can run a fit.  There are two buttons at the bottom of the
+panel: *Qick fit* and *Full fit*. One should always start with the
+*Quick fit*. The only difference between the two is that the first will only
+run the calculation once and produce the result.
+
+IMAGE INSERTED HERE OF QUICK FIT
+
+After a short time, the graph will be updated with the fit to the data using
+the resulting distribution, while a second plot will pop up showing the final
+distribution of sizes that are returned, giving the volume fraction (true or
+relative depending on whether the data are on absolute scale or not) of each
+size. Finally the *Ouput* section of the *Paremeters* tab will show the
+results including whether or not the fitting converged, the unormalized
```

Comment:
normalized? unnormalized?

Reply:
A good question. Probably normalized but need to look at the code. I think I thought it was unnormalized because of the very high chi^2 I was seeing but actually playing around with it more I think that it may in fact be a correct value for normalized!  Will check and correct if appropriate. Thanks!

Reply:
And the answer is..... in fact what seems to be returned is the unnormalized chi^2 though it is the normalized chi^2 that is used to test for convergence. I do not understand why they do this? It would be pretty easy to return the normalized chi^2 I think.  For now I will leave the documentation like this.

However, if and when we change the behavior we should not forget to change the documentation!